pkg/framework/response.go

```diff
@@ -53,8 +56,27 @@ func (r *Response) SetSuccess(flag bool) {
 	r.success = flag
 }
 
-func (r *Response) Error(err error) {
+func (r *Response) Error(err error, code int) {
 	r.err = err
+
```

Comment:
why are you hardcoding the paths to the views in the framework package?

Reply:
these templates are part of the package `app`.if you do this you cant use `framework` package in other apps